src/Sarif.Multitool.Library/KustoCommand.cs

```diff
@@ -130,15 +130,24 @@ private void InitializeKustoClient()
                         onDemandValidated = (sbyte)dataReader.GetValue(GetIndex(dataReader, dataReaderIndex, "OnDemandValidated")) == 1;
                     }
 
-                    if (etlEntity == "Build" || etlEntity == "Release" || etlEntity == "BuildDefinition" || etlEntity == "ReleaseDefinition")
+                    if (etlEntity == "Build" || etlEntity == "BuildDefinition" ||
+                        etlEntity == "Release" || etlEntity == "ReleaseDefinition")
                     {
                         itemPath = itemPath.Replace("vsrm.visualstudio.com", "visualstudio.com");
                         itemPath = itemPath.Replace("_apis/build/Definitions/", "_build?definitionId=");
                         itemPath = itemPath.Replace("_apis/Release/definitions/", "_release?_a=releases&view=mine&definitionId=");
+                        itemPath = itemPath.Replace("_apis/wit/workItems/", "_workitems/edit/");
 
                         resultMessageText += $" This pipeline can be updated on [Azure DevOps]({itemPath}) to secure the variable that exposes this secret.";
                     }
 
+                    if (etlEntity == "WorkItem")
+                    {
+                        itemPath = itemPath.Replace("_apis/wit/workItems/", "_workitems/edit/");
+
+                        resultMessageText += $" This work item can be updated on [Azure DevOps]({itemPath}) to secure the variable that exposes this secret.";
```

Comment:
>resultMessageText [](http://example.com/codeflow?start=24&length=17)

why are you adding this text? i don't think we should. 

also, what does it mean to 'secure the variable' on a work item? i *think* you can only redact a secret from a work item (and that doesn't actually solve the problem, you have to invalidate it).

Reply:
The idea is to add a direct link to the workitem instead of redirecting to the json file.
I removed the part to "secure the variable".